src/lib/viewers/BaseViewer.js

```diff
@@ -1081,6 +1081,30 @@ class BaseViewer extends EventEmitter {
         this.annotator.toggleAnnotationMode(AnnotationMode.NONE);
     }
 
+    /**
+     * Handler for annotation mode change
+     * 1. Set annotationsControls mode
+     * 2. If discoverability FF is on, add mode classes to container
+     *
+     * @param {AnnotationMode} mode Next annotation mode
+     */
+    handleAnnotationModeChange = mode => {
+        if (this.annotationControls) {
+            this.annotationControls.setMode(mode);
+        }
+
+        if (this.options.enableAnnotationsDiscoverability) {
```

Comment:
This is unrelated to this change, but should we check to see if `this.containerEl` exists?